src/Toolkit/Toolkit/UI/Controls/BookmarksView/BookmarksViewDataSource.cs

```diff
@@ -91,7 +91,7 @@ public void SetOverrideList(IEnumerable<Bookmark> bookmarks)
             // Subscribe to events if applicable
             if (bookmarks is INotifyCollectionChanged iCollectionChanged)
             {
-                var listener = new Internal.WeakEventListener<INotifyCollectionChanged, object, NotifyCollectionChangedEventArgs>(iCollectionChanged);
+                var listener = new Internal.WeakEventListener<INotifyCollectionChanged, object?, NotifyCollectionChangedEventArgs>(iCollectionChanged);
```

Comment:
Side note: I just noticed that our WeakEventListener pattern is wrong in the toolkit, in the same way that it used to be wrong in dotnet-api. Most current uses of WeakEventListener actually create strong listeners. I need to port the fixes here some time...